Modules/_asynciomodule.c

```diff
@@ -3944,6 +3947,8 @@ static int
 module_exec(PyObject *mod)
 {
     asyncio_state *state = get_asyncio_state(mod);
+
+    _Py_SetImmortal((PyObject *)&state->asyncio_tasks.tail);
```

Comment:
Oh, it might need a valid `ob_type` or some assertions will break.

Reply:
Fixed